Configurations.md

```diff
@@ -1063,7 +1063,7 @@ The pattern format is the same as [.gitignore](https://git-scm.com/docs/gitignor
 
 - **Default value**: format every file
 - **Possible values**: See an example below
-- **Stable**: No (tracking issue: [#3395](https://github.com/rust-lang/rustfmt/issues/3395))
+- **Stable**: Yes
```

Comment:
Do you think we should fix #3881 before stabilizing this? Or shall we go ahead and stabilize but treat #3881 as bug?

Reply:
Good catch. I will fix that first. Thanks!

Reply:
#3881 is fixed by #4179.